src/containers/HomeV2/RecommendationListCard.jsx

```diff
@@ -66,41 +66,38 @@ class RecommendationListCard extends React.PureComponent {
 
     // temporary detection for CodeCombat Activities,
     // to change the avatar, logo, and color
-    const isCodeCombat = (title==="CodeCombat Activities")
-      ? true
-      : false;
+    const isCodeCombat = title === "CodeCombat Activities";
 
     return (
-      <div style={{marginBottom : "24px"}}>
-      {/* either use marginBottom here or put a <br /> at parent */}
+      <div style={{ marginBottom: "24px" }}>
+        {/* either use marginBottom here or put a <br /> at parent */}
         <Card style={styles.card}>
-          {(RecomType === "python") &&
+          {RecomType === "python" && (
             <Fragment>
               {/* this is temporarily testing the GUI with CodeCombat type
                   feel free to refactor it to a proper solution */}
               <CardHeader
                 avatar={
-                  <Avatar aria-label="Recommendation"
-                    style={isCodeCombat
-                      ? styles.avatarCC
-                      : styles.avatarPy}
+                  <Avatar
+                    aria-label="Recommendation"
+                    style={isCodeCombat ? styles.avatarCC : styles.avatarPy}
                   >
                     {isCodeCombat ? <Adb /> : "</>"}
                   </Avatar>
                 }
                 subheader="Recommended for you"
-                title={title|| "Jupyter Notebook Activities"}
+                title={title || "Jupyter Notebook Activities"}
               />
               <CardContent>
                 <SampleCarousel
                   dataList={dummyData}
-                  youtubeRecom={false}
                   isCodeCombat={isCodeCombat}
```

Comment:
@Brainenjii `isCodeCombat` was inserted as temporary fix for the GUI to render a row for CodeCombat type activities. There should be some refactoring to expand the detection of activity types.